dotCMS/src/integration-test/java/com/dotcms/contenttype/test/ContentResourceTest.java

```diff
@@ -921,6 +1050,34 @@ private void validateChildrenXML(final Map<String, Contentlet> contentletMap,
         }
     }
 
+    private void validateCategoryXML(
+            final String resultXml, final String categoryElement, final Category... categories)
+            throws XPathExpressionException {
+
+        final InputSource xmlSource = new InputSource(new StringReader(
+                resultXml.replaceAll("\\n", "")));
+
+        XPath xPath = XPathFactory.newInstance().newXPath();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'xPath' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4885353)